### PR TITLE
Fix some regressions in Powerline theme

### PR DESCRIPTION
**Pull request by edubxb**
Fix some regressions introduced in PR https://github.com/Bash-it/bash-it/pull/891.

### CODE REVIEW

File: themes/powerline/powerline.base.bash [hunk 1]
```diff
@@ -14,19 +14,24 @@ function set_color {
...
-        color=${USER_INFO_THEME_PROMPT_COLOR_SUDO}
-        user_info=${USER_INFO_SUDO_CHAR}${USER}
+      if [[ "${color}" = "${USER_INFO_THEME_PROMPT_COLOR_SUDO}" ]]; then
+        user_info="!"
       fi
       ;;
     *)
       if [[ -n "${SSH_CLIENT}" ]] || [[ -n "${SSH_CONNECTION}" ]]; then
-        user_info="${USER_INFO_SSH_CHAR}${USER}"
+        user_info="${USER_INFO_SSH_CHAR}${USER}@${SHORT_HOSTNAME:=$HOSTNAME}"
```
Comment (dnoiz1) [inline themes/powerline/powerline.base.bash]: imo the hostname is already an element of powerline, forcing it into the user info removes the users ability to customise it in their .bashrc

Comment (edubxb) [reply themes/powerline/powerline.base.bash]: Yeah, just noticed the new function (`__powerline_hostname_prompt`), but, there's no way to only show the hostname when connected to a remote host, like the old implementation does.

I will add another variable to control the behavior, what do you think?

**Comment by nwinkler** [comment]
@edubxb Is this ready to merge now? Or are you still working on this?

**Comment by edubxb** [comment]
Is ready, I will create another PR probably, thanks!

**Comment by nwinkler** [comment]
Thanks! Merged...

**Pull request merged**